README.md

```diff
@@ -330,6 +330,22 @@ skip_string_normalization = true
 
 For more information check the `super-linter` readme.
 
+### Pre-commit
+
+[Pre-commit](https://pre-commit.com/) is a framework for managing git pre-commit hooks. Using this framework you can run `snakefmt` whenever you commit a `Snakefile` or `*.smk` file. `Pre-commit` automatically creates an isolated virtual environment with `snakefmt` and will stop the commit if `snakefmt` needed to modify the file. You then review, stage, re-commit these changes. Pre-commit is especially useful if you don't have access to a CI/CD systems like GitHub actions.
+
+To do so, create the file `.pre-commit-config.yaml` in the root of your project directory with the following:
+
+```yaml
+repos:
+-    repo: https://github.com/snakemake/snakefmt
+     rev: 0.2.4 # Replace by any tag/version ≥0.2.4 : https://github.com/snakemake/snakefmt/releases
```

Comment:
```suggestion
     rev: 0.2.3 # Replace by any tag/version: https://github.com/snakemake/snakefmt/releases
```

Reply:
@mbhall88 doesn't it need to be >=0.2.4 (if we release this in 0.2.4) as prior to that `pre-commit` will not have access the pre-commit-hooks.yaml file @jfear is adding?

Reply:
@bricoletc is correct, pre-commit would look at the tagged version which would not have the config file. You can point directly to a commit hash or to a branch (i.e., `master`) instead of a tag. However, I would think most users would want to use a specific tagged version. 

I am fine with merging into dev and then just including in the next release.

Reply:
Great- in that case we can leave that part of the text referring to >=0.2.4, i agree with referring to releases. 
@jfear can you update the text with @mbhall88 's suggestions (eg needed to --> would) on your branch? Then we can this merge to dev (assuming running pre-commit on your branch works?)

Unless you ask otherwise, we'd like to acknowledge your help in the CHANGELOG for next release. Thanks!

Reply:
Ok, I am going to leave this as is then unless you want me to change.

Reply:
Ah, yes, you are both correct.